symphony/bdk/ext/group.py

```diff
@@ -108,6 +111,30 @@ async def list_groups(self, status: Status = None, before: str = None, after: st
             kwargs["sort_order"] = sort_order
         return await self._group_api.list_groups(**kwargs)
 
+    @retry(retry=refresh_bearer_token_if_unauthorized)
```

Comment:
`@retry` is not needed here because we already have it for the `list_groups` method

Reply:
Done in https://github.com/finos/symphony-bdk-python/pull/276/commits/a8d37738b59949adb32c2dd7dcf53eee730a2d28